includes/templates/class-amp-content.php

```diff
@@ -184,7 +186,9 @@ private function register_embed_handlers( $embed_handler_classes ) {
 				continue;
 			}
 
-			$embed_handler->register_embed();
+			if ( $embed_handler instanceof Registerable ) {
```

Comment:
Needed for backwards-compatibility:

```suggestion
			if ( $embed_handler instanceof Registerable || method_exists( $embed_handler, 'register_embed' ) ) {
```